.github/workflows/validate_packs.yml

```diff
@@ -21,3 +21,25 @@ jobs:
       - name: Validate files
         id: validate-files
         run: yarn validate-packs 
+
+  ensure-pack-names-are-unique:
+    name: Ensure pack names are unique
+    runs-on: ubuntu-latest
+    needs: validate-packs
```

Comment:
We might not want to have this. As a contributor, I would like to see _all_ the reasons my pack was invalid when it goes through the validation workflow.